test/TestCliRolePaths.py

```diff
@@ -90,12 +73,30 @@ def test_run_playbook(self):
         env = os.environ.copy()
         env['ANSIBLE_ROLES_PATH'] = os.path.dirname(cwd)
 
-        result = self.run_ansible_lint(cwd=cwd, role_path=role_path, env=env)
+        result = run_ansible_lint(cwd=cwd, role_path=role_path, env=env)
         self.assertIn('Use shell only when shell functionality is required', str(result))
 
     def test_run_role_name_invalid(self):
         cwd = self.local_test_dir
         role_path = 'roles/invalid-name'
 
-        result = self.run_ansible_lint(cwd=cwd, role_path=role_path)
+        result = run_ansible_lint(cwd=cwd, role_path=role_path)
         assert '106 Role name invalid-name does not match' in str(result)
+
+
+def test_run_playbook_github():
+    """Call ansible-lint simulating GitHub Actions environment."""
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    role_path = '../examples/example.yml'
+
+    env = {
+        "GITHUB_ACTIONS": "true",
+        "GITHUB_WORKFLOW": "foo"
+    }
```

Comment:
We need a negative test where these values don't match the condition. You could use parameterized to keep them all in one test function.